package.json

```diff
@@ -79,8 +80,11 @@
     "node": ">=18"
   },
   "workspaces": [
-    "app/*",
```

Comment:
interesting that this worked with pnpm

my guess is this is such a common mistake they just decided to support it - maybe we'll do something similar